src/d3d11/d3d11_fence.cpp

```diff
@@ -66,6 +95,11 @@ namespace dxvk {
 
 
   UINT64 STDMETHODCALLTYPE D3D11Fence::GetCompletedValue() {
+    if (m_flags & D3D11_FENCE_FLAG_SHARED) {
+      // TODO directly call vkGetSemaphoreCounterValue here, getValue returns the monotonic tracked value
```

Comment:
Same here, even if a fence is rewound then the reported fence value will be updated within 10ms.

We can improve this behaviour in the backend if necessary, but disabling this function seems unnecessary unless there is an extremely good reason to.